.github/STALE-BOT.md

```diff
@@ -0,0 +1,36 @@
+# Stale bot information
+
+- Thank you so much for your contribution!
+- If you would like the bot to remove the label, simply leave a comment.
+- _How to find the right people to ping?_ &rarr; [`git blame`](https://git-scm.com/docs/git-blame) to the rescue or GitHub's history button!
+- You can always ask for help on [our Discourse Forum](https://discourse.nixos.org/) or on the [#nixos' IRC channel](https://webchat.freenode.net/#nixos).
+
+## PR author suggestions
+
+1. If it is unfinished and you plan on finishing it, please mark it as a draft.
+2. If you want to abandon this work, please consider closing it with a short comment. This might encourage somebody else to pick up your work.
+3. If you want to get things rolling again, first rebase the PR and address all valid comments.
+4. Then, if you need a review to push things forward, [try this Discourse thread](https://discourse.nixos.org/t/prs-in-distress/3604).
+5. If all you need is a merge, check the git history to find and [request reviews](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/requesting-a-pull-request-review) from people who usually merge related contributions.
+
+## Issue author suggestions
+
+1. If it is resolved (either for you personally, or in general), please consider closing it.
+2. If this might still be an issue, but you are not interested in promoting its resolution, please consider closing it while encouraging others to take over and reopen an issue if they care enough.
+3. If you still have interest in resolving it, try to ping somebody who you believe might have an interest in the topic. Consider discussing the problem in [our Discourse Forum](https://discourse.nixos.org/).
+4. As with all open source projects, your best option is to submit a Pull Request that addresses this issue. We :heart: this attitude!
```

Comment:
I'm still chewing on this part. I feel a little like something is missing or maybe the order is just off? But I'm feeling a little dull at the end of the day, and this strikes me as one of the most-important sections (given the quantity of issues). 

Just flagging it so that I can send along what I've got already and break for the evening. :)

Reply:
I missed your first comment about merging tomorrow; I'll try to find something bright to say in the morning ;)

Reply:
It was just an arbitrary deadline to give you some time to respond. I can wait longer. Thanks for your input!

Reply:
I think I'm reacting to this section being a little vague, but it probably does have to be a little vague to cover the different kinds of issue and reasons it may be open.

I'm not sure this thought should hold it up, here. I don't feel compelled to rephrase anything, and I suspect it would be hard to address without turning this a flowchart where we can give more situational recommendations without overwhelming people? I sketched out something like the below on a piece of paper:

- what kind of issue is this?
   - bug/break
      - can you confirm it still exists in (nixpkgs unstable? master?)?
         - no, it doesn't
            - ok--please close it
         - no, I can't (I no longer use Nix, or NixOS, or this specific package? I don't have time/interest?)
            - ok--please close it, but mention anyone else who chimed in to say "me too" so that they can open an issue if they want.
         - yes, I can
            - ok, please comment confirming you still see the problem with a fresh copy of the output from `nix-info -m`
            - here are a few more ways you can help us resolve this issue faster
               - give solving the issue a try! someone on IRC or discourse may be able to help you if you're stuck...
               - add a test that will break until this issue is fixed...
               - advocacy instructions: how to fish around for interested parties on GH and/or discourse
   - suggestion/request (package request, version updates, ...?)
      - do you still want/need/care about this?
         - yes
            - suggest pitching in
            - advocacy instructions
         - no
            - please close it, encourage others who indicated interest to take it up

Reply:
Sorry to make you wait for a ~no-op! :)